src/pybind/mgr/dashboard/frontend/src/app/ceph/nfs/nfs-form/nfs-form.component.ts

```diff
@@ -213,7 +223,7 @@ export class NfsFormComponent extends CdForm implements OnInit {
       if (subvolGroup === this.defaultSubVolGroup) {
         this.updatePath('/volumes/' + this.defaultSubVolGroup);
         resolve();
-      } else {
+      } else if(subvolGroup != ''){
```

Comment:
We must add comments here as the logic in this file is complex.
Lets add some useful comments why we are doing this. Thanks